lib/routes/ncu/jwc.ts

```diff
@@ -46,12 +46,21 @@ async function handler() {
                 const date = el.text().split('】')[0].replace('【', '').trim();
                 const title = linkEl.attr('title');
                 const link = `${baseUrl}/${linkEl.attr('href')}`;
-                const month = date.slice(0, 2);
+
+                const newsDate = parseDate(date, 'MM-DD');
+                const currentYear = new Date().getFullYear();
+                const currentMonth = new Date().getMonth() + 1; // getMonth() returns month index starting from 0
+                const newsMonth = newsDate.getMonth() + 1;
```

Comment:
Don't `new` 2 `Date` objects in the map loop. Create only one outside the loop is sufficient.
```ts
    const currentDate = new Date();
    const currentYear = currentDate.getFullYear();
    const currentMonth = currentDate.getMonth() + 1;
    ...
    list.toArray().map((item) => {
        ...
        const newsDate = parseDate(date, 'MM-DD');
        const newsMonth = newsDate.getMonth() + 1;
        ...
```